.yamato/config/abv.metafile

```diff
@@ -28,21 +28,6 @@ jobs:
         allow_on:
           - 2021.2
 
-  - name: Trunk Weekly
-    job_id: all_project_ci_trunk_weekly
-    dependencies:
-      - .yamato/all-hdrp.yml#Nightly_HDRP_<TRACK>
-      - .yamato/all-vfx.yml#Nightly_VFX_<TRACK>
-      - .yamato/all-urp.yml#Nightly_URP_<TRACK>
-      - .yamato/all-shadergraph.yml#Nightly_ShaderGraph_<TRACK>
-      - <PROJECT_ALL_YML_FILE>#General_Checks_<TRACK>
-    triggers:
-        recurring:
-          -  branch: master
```

Comment:
So that we don't forget, we also need to update all the triggers `branch: master` to `2021.2/staging`. We might be using `master` references in other contexts as well, but I'm not sure, need to doublecheck this